README.md

```diff
@@ -33,9 +33,18 @@ cd app
 npm install
 ```
 
-### Enable the Simulator (Optional)
+### Setup Script
+
+For quick setup, run `quick_setup_script.sh` with `MASTER_WALLET_PK` set as an environment variable to the master private key found in `/config_files/secret/local_secrets.ini/`.
+
+```
+quick_setup_script.sh [activation path for your python env]
+```
```

Comment:
This kinda makes the ###Blockchain section redundant. Probably need to rewrite the readme

Reply:
I'll tweak it - should only be a small change.